lms/djangoapps/certificates/management/commands/update_cert_status.py

```diff
@@ -0,0 +1,111 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+from __future__ import print_function
+
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import CertificateStatuses
+from certificates.models import GeneratedCertificate
+
+LOGGER = logging.getLogger(__name__)
+
+VALID_STATUSES = [
+    CertificateStatuses.unavailable,
+    CertificateStatuses.downloadable,
+    CertificateStatuses.notpassing,
+]
+
+
+class Command(BaseCommand):
+    """
+    Update the status of a single user certificate for a given user in a given course
+    """
+    help = __doc__
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course_id',
+            default=False,
+            help='The course id (e.g. mit/6-002x/circuits-and-electronics or course-v1:edX+DemoX+Demo_Course)'
+                 ' of the course in which the certificate of the student should be updated',
+        ),
+        make_option(
+            '-u', '--username_or_email',
```

Comment:
I don't know that I've ever seen command line args with underscores;
words should be separated with hyphens; `--username-or-email`